lib/matplotlib/tests/test_fontconfig_pattern.py

```diff
@@ -1,7 +1,7 @@
-import pytest
```

Comment:
Since `pytest` is a third party library, it was above the matplotlib import following [the PEP8 style guide](https://peps.python.org/pep-0008/#imports) so ideally it should stay there.